src/base/utils/fs.cpp

```diff
@@ -267,9 +273,7 @@ bool Utils::Fs::isValidFileSystemName(const QString &name, const bool allowSepar
 
 qint64 Utils::Fs::freeDiskSpaceOnPath(const QString &path)
 {
-    if (path.isEmpty()) return -1;
-
-    return QStorageInfo(path).bytesAvailable();
+    return QStorageInfo {path}.bytesAvailable();
```

Comment:
This is exactly what I meant when I talked about the exceptions [there](https://github.com/qbittorrent/qBittorrent/pull/14054#discussion_r556316393). It looks confusing, IMO. We should either keep using parentheses in similar cases or wrap all initialization in parentheses:
```c++
return QStorageInfo(path).bytesAvailable();
// or
return (QStorageInfo {path}).bytesAvailable();
```

Reply:
I chose the first one.

Reply:
Or maybe rethink entire rule for initialization? Drop `space` requirement:
```c++
return QStorageInfo{path}.bytesAvailable();
```

Reply:
>Drop space requirement:

After getting used of having space before brackets, now it looks a little weird..

Reply:
>After getting used of having space before brackets, now it looks a little weird.. 

IMO, it looks good in any cases except this one (if member is accessed without creating variable). `QStorageInfo {path}.bytesAvailable()` breaks the relationship between the elements of expression for my eyes. So any of mentioned above notation is better for me.

Reply:
Well, now I'm faced with a case where we also have to use parentheses for initialization (and unlike the case above, it's not just about the coding style, so there are no other options).
If the class has a constructor that accepts the `std::initializer_list` as a parameter, then this particular overload will be selected if we use uniform initialization. So if we need to call another constructor, we have to use the syntax with parentheses.
For example, if I want to create a QVector of N elements with the value M, then:
```c++
// int N; Type M;
QVector<Type> vec {N, M}; // Wrong! Creates [N, M] if N is convertible to Type. If N isn't convertible to Type I don't know how it behaves exactly, maybe implementation dependent.

QVector<Type> vec(N, M); // Valid! Creates [M, M, ..., M] of size N.
```